src/game/InstanceData.h

```diff
@@ -104,10 +104,10 @@ class LOOKING4GROUP_IMPORT_EXPORT InstanceData : public ZoneScript
         virtual void ResetEncounterInProgress();
 
         //Called when a player successfully enters the instance.
-        virtual void OnPlayerEnter(Player *) {}
+        void OnPlayerEnter(Player *);
```

Comment:
Yeah, bit of a nightmare. Been trying to come up with elegant solutions but it seems I'll have to go through those examples (at least those I want to log the player ids of those who enter the instance) and call the base class implementation of `OnPlayerEnter` at the top of the override.
Also applies to `OnPlayerDeath` in various scenarios.